db/migrate/20230516113713_create_products.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProducts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :products, id: :uuid do |t|
+      t.string :name
+      t.text :description
+      t.decimal :price
+      t.string :image
```

Comment:
Could these information be nil?

Reply:
right now i've not placed any validators over any variables or any sort of data constraints, if you could allow me to work more on this code then i can certainly give you a better version, just so that there was a lot to be done and commit time was just 1hr.